include/zephyr/drivers/video.h

```diff
@@ -732,19 +732,21 @@ static inline int video_set_signal(const struct device *dev, enum video_endpoint
  *
  * @param size Size of the video buffer (in bytes).
  * @param align Alignment of the requested memory, must be a power of two.
+ * @param timeout How long to wait, or K_NO_WAIT.
```

Comment:
I suggest rephrasing this to:
`@param timeout Timeout duration in milliseconds or K_NO_WAIT`

Reply:
If it is a [`k_timeout_t`](https://docs.zephyrproject.org/apidoc/latest/structk__timeout__t.html), would the unit depend on which macro is used? `K_MSEC()`, `K_USEC()`...

Reply:
Oh, true. My intention is to remove the "How long to wait" from there. Agrre that "milliseconds" isn't correct.